app/src/main/java/com/android/brewr/ui/overview/Map.kt

```diff
@@ -26,61 +21,34 @@ import com.google.maps.android.compose.rememberCameraPositionState
 import kotlinx.coroutines.tasks.await
 
 @Composable
-fun MapScreen(listLocations: List<Location>) {
+fun MapScreen(coffees: List<Coffee>) {
   val context = LocalContext.current
   var userLocation by remember { mutableStateOf<LatLng?>(null) }
-  var permissionGranted by remember { mutableStateOf(false) }
 
-  val locationPermissionLauncher =
-      rememberLauncherForActivityResult(
-          contract = ActivityResultContracts.RequestMultiplePermissions(),
-          onResult = { permissions ->
-            permissionGranted =
-                permissions[Manifest.permission.ACCESS_FINE_LOCATION] == true ||
-                    permissions[Manifest.permission.ACCESS_COARSE_LOCATION] == true
-          })
-
-  LaunchedEffect(Unit) {
-    permissionGranted =
-        ContextCompat.checkSelfPermission(context, Manifest.permission.ACCESS_FINE_LOCATION) ==
-            PackageManager.PERMISSION_GRANTED ||
-            ContextCompat.checkSelfPermission(
-                context, Manifest.permission.ACCESS_COARSE_LOCATION) ==
-                PackageManager.PERMISSION_GRANTED
-
-    if (!permissionGranted) {
-      locationPermissionLauncher.launch(
-          arrayOf(
-              Manifest.permission.ACCESS_FINE_LOCATION, Manifest.permission.ACCESS_COARSE_LOCATION))
-    }
-  }
-
-  LaunchedEffect(permissionGranted) {
-    if (permissionGranted) {
-      userLocation = getCurrentLocation(context)
-    }
-  }
+  LaunchedEffect(Unit) { userLocation = getCurrentLocation(context) }
 
   Scaffold(
       content = { paddingValues ->
         val cameraPositionState = rememberCameraPositionState {
-          position = CameraPosition.fromLatLngZoom(userLocation ?: LatLng(37.7749, -122.4194), 10f)
+          position = CameraPosition.fromLatLngZoom(userLocation ?: LatLng(46.5197, 6.6323), 14f)
         }
 
         GoogleMap(
             modifier = Modifier.fillMaxSize().padding(paddingValues).testTag("mapScreen"),
             cameraPositionState = cameraPositionState) {
-              listLocations.forEach { location ->
+              coffees.forEach { coffee ->
                 Log.d(
                     "MapScreen",
-                    "Adding marker for ${location.address} at (${location.latitude}, ${location.longitude})")
+                    "Adding marker for ${coffee.location.address} at (${coffee.location.latitude}, ${coffee.location.longitude})")
                 Marker(
```

Comment:
I saw that, for example, for "Mc donald" and a random coffee shop, they have the same marker icon. I think for later sprints, it would be nice if we try to clasify / differentiate coffeeshops and others with coffee icon marker and else. This is just an idea for future sprints, anyways well done !

Reply:
Yeah it would be a good idea!